similar_dev_search/model/repository.py

```diff
@@ -36,3 +41,40 @@ async def get_stargazers(self, asyncio_client: httpx.AsyncClient = None) -> List
         await client.aclose()
         return developers_urls
 
+    async def _add_file_info(self, author_id: str, file: ModifiedFile) -> None:
```

Comment:
right now it should give number of added/deleted/modified lines in specific file to "experience" of developer